wp-graphql.php

```diff
@@ -280,11 +276,9 @@ public function setup_types() {
 			/**
 			 * Setup the settings, post_types and taxonomies to show_in_graphql
 			 */
-			\WPGraphQL::show_in_graphql();
-			\WPGraphQL::get_allowed_post_types();
-			\WPGraphQL::get_allowed_taxonomies();
-
-
+			self::show_in_graphql();
```

Comment:
@jasonbahl I do wonder why it was calling itself that way. Any particular reason?

Reply:
no reason that I can recall 🤔 

I think at one point they were in different classes or something. I don't recall. But this makes more sense!